src/w3d/math/gamemath.h

```diff
@@ -547,4 +530,31 @@ inline int Lrintf(float val)
 //    return true;
 //}
 
+inline bool Is_Nan(float val)
+{
+#ifdef BUILD_WITH_GAMEMATH
+    return gm_isnanf(val);
```

Comment:
```
D:\a\Thyme\Thyme\src\w3d\math\gamemath.h(536): error C4800: Implicit conversion from 'int' to bool. Possible information loss
D:\a\Thyme\Thyme\src\w3d\math\gamemath.h(536): note: consider using explicit cast or comparison to 0 to avoid this warning
D:\a\Thyme\Thyme\src\w3d\math\gamemath.h(561): fatal error C1004: unexpected end-of-file found
```

Reply:
I compile with MSVC v140 and no such errors,.

Reply:
See build below. Compile errors on some targets.